pkgs/by-name/bl/bleep/package.nix

```diff
@@ -0,0 +1,89 @@
+{
+  stdenv,
+  fetchurl,
+  autoPatchelfHook,
+  installShellFiles,
+  makeWrapper,
+  lib,
+  zlib,
+}:
+let
+  platform =
+    {
+      x86_64-linux = "x86_64-pc-linux";
+      x86_64-darwin = "x86_64-apple-darwin";
+      aarch64-darwin = "arm64-apple-darwin";
+    }
+    ."${stdenv.system}";
+
+  hash =
+    {
+      x86_64-linux = "sha256-ttfmTMSK6u2zX87CUYjibr675Xzn4/byWub67e5Ir+I=";
+      x86_64-darwin = "sha256-bh1yQ7KZ+cwVHTLG+gpYDLPKG2VUcrz88Sqd2tRiKfM=";
+      aarch64-darwin = "sha256-IV2JPXibv80Odb+1Zkw1YUoutHsi5YaIjyNpAOIUlr4=";
+    }
+    ."${stdenv.system}";
+in
+stdenv.mkDerivation rec {
```

Comment:
Use the `finalAttrs` pattern instead of `rec`: https://nixos.org/manual/nixpkgs/stable/#mkderivation-recursive-attributes